tests/x509/test_x509_ext.py

```diff
@@ -6170,6 +6154,23 @@ def test_public_bytes(self, backend):
             b"\x85\x1f%\xc0#N\\\xdeK\x90d\xe0\x8d<{\xca\xdbdc\xeft\x87g\x10"
         )
 
+    def test_strip_scts(self, backend):
```

Comment:
The name should mention `tbs_precertificate_bytes` and probably should be next to the other test

Reply:
Okay, moved.